src/config.js

```diff
@@ -52,27 +52,58 @@ export const profiles = {
 //   ]
 // }
 
+export const intro = {
+  firstTitle: "Social",
+  secondTitle: "Warming",
+  text: "Op deze pagina staat vastgelegd hoe de groep social warming te werk gaat met de design challenge : ‘Maak een digitale toepassing over het thema social cooling voor Bevrijdingsdag 5 mei 2020.’. Op deze onepager staat ons creatief proces vastgelegd."
+}
+
 export const progress = {
   designTactics: {
     firstTitle: "Ontwerp",
     secondTitle: "technieken",
-    intro: "Onderstaande technieken hebben we gebruikt om te ontwerpen.",
+    intro: "Dit is een overzicht van alle creatieve technieken die we hebben gebruikt om onze concepten te ontwikkelen.",
     bgColor: "black",
     contentBlocks: [
       {
         content: {
-          title: "Eerste taak",
+          title: "Mindmap",
           subtitle: "Optioneel?",
-          text: "Dit is de eerste taak die uitgevoerd is. Top!"
+          text: "Mindmapping is een creatieve techniek die helpt bij het leggen van connecties bij een onderwerp. In de mindmap staat een onderwerp centraal. Dit onderwerp breed je uit met verschillende takken, deze takken zijn woorden die je associeert bij het onderwerp. Deze takken worden nu het onderwerp van de volgende vertakkingen. Zo heb je uiteindelijk een heel web met begrippen sommige hiervan hebben met het hoofdonderwerp te maken en sommige ook niet. Wij hebben deze methode gebruikt omdat het een overzicht van het onderwerp geeft en alle onderdelen hier erbij hoorden. En tijdens het maken van de mindmapping kom je er achter als groep waar je nog dieper op in moet gaan.",
+          img: "./img/mindmapping",
```

Comment:
Als het goed is stond er een voorbeeld van een img Object in, klopt dit? Zo ja, kan je deze dan nog aanpassen